todo_app/templates/index.html

```diff
@@ -22,11 +22,21 @@ <h2>Items</h2>
   <form action="http://127.0.0.1:5000/add" method="get" class="form-example">
     <div class="form-example">
       <label for="title">Enter the title: </label>
-      <input type="text" name="title" id="title" required>
+      <input type="text" name="title" id="title" >
     </div>
     
     <div class="form-example">
-      <button type="submit" formmethod="post">Add Item</button>
+      <button type="submit" formmethod="post" name="add_button" value="Add">Add Item</button>
+    </div>
+  </form>
+  <form action="http://127.0.0.1:5000/complete_item" method="get" class="form-example">
```

Comment:
It looks like you've added a link directly to each item that can complete items, so we should remove this less-user friendly route. That would also allow us to remove the explicit printing out of the trello card/list ids